app/locales/en.json

```diff
@@ -47,7 +47,7 @@
   "Join us in the most important crowdsource!": "Join us in the most important crowdsource!",
   "Make your voluntary contribution to the health care system and register your health condition now. Together we can create an overview.": "Make your voluntary contribution to the health care system and register your health condition now. Together we can create an overview.",
   "Share on": "Share on",
-  "NB!": "NB!",
+  "NB!": "Important!",
```

Comment:
Should we have a trailing colon here?

Reply:
Either a trailing colon or the exclamation point, as-is, but not both.

Reply:
Ah my bad, I misread, exclamation point should do!